2-collectors/scala-stream-collector/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -18,14 +18,12 @@ package collectors.scalastream
 import java.util.UUID
 
 import scala.collection.JavaConverters._
-
 import akka.http.scaladsl.model._
 import akka.http.scaladsl.model.headers._
 import akka.http.scaladsl.model.headers.CacheDirectives._
 import org.apache.commons.codec.binary.Base64
 import org.slf4j.LoggerFactory
 import scalaz._
-
```

Comment:
we tend to separate imports like this:

- java
- scala
- 3rd party
- 1st party

Reply:
Done.